pkg/v1/layout/index.go

```diff
@@ -126,6 +126,19 @@ func (i *layoutIndex) Blob(h v1.Hash) (io.ReadCloser, error) {
 	return i.path.Blob(h)
 }
 
+// Workaround for #819.
```

Comment:
I noticed this comment in other locations where the private interfaces exposing extra extensibility are implemented